app/updatemanager.go

```diff
@@ -459,6 +459,27 @@ func (c *ControlMapPool) ClearExpired() {
 	c.saveToStore()
 }
 
+// Removes maps from the pool for which ID match deploymentId
+func (c *ControlMapPool) ClearIdMatchingDeploymentId(deploymentId string) {
```

Comment:
How is this different from this: https://github.com/mendersoftware/mender/blob/d2c66f51a2e336337f0ee9daf133eeb3dba7b9ab/app/updatemanager.go#L421

:thinking:

Reply:
In that mine is documented? :laughing: 

Nah, I wrote this long ago (predates `Delete`) and I didn't realize while rebasing. Removing it now...

Reply:
Not only mine is documented, but unit tested! Which I see is not the case for `Delete`...